src/appengine/handlers/cron/bandit_probabilities.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bandit probabilities cron job. Queries from bigquery for updated probability
+	values based on new edges for various combined strategies."""
+
+from datastore import data_types
+from datastore import ndb
+from google_cloud_utils import big_query
+from handlers import base_handler
+from libs import handler
+
+MULTIPLE_STRATEGY_PROBABILITY_QUERY = """
```

Comment:
I still plan to review this in more depth. Do you have any docs for what you based these calculations on to save me some time? It may also be worth referencing them in a comment.